lib/src/base-styles/_nx-page-header.scss

```diff
@@ -39,61 +39,44 @@ $main-header-product-version-row-height: 1fr;
 .nx-product {
   align-self: center;
   box-sizing: border-box;
-  color: $nx-off-white;
-  display: -ms-grid;
-  display: grid;
-  grid-column-gap: $nx-spacing-xxs;
-  -ms-grid-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
-  grid-template-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
-  -ms-grid-columns: $main-header-product-height $nx-spacing-xxs auto;
-  grid-template-columns: $main-header-product-height auto;
-  grid-template-areas:
-    "logo title"
-    "logo version";
+  color: $main-header-color;
+  display: flex;
   height: $main-header-product-height;
   text-decoration: none;
   min-width: $nx-sidebar-width;
 }
 
 .nx-product__logo {
-  -ms-grid-row: 1;
-  -ms-grid-row-span: 2;
-  -ms-grid-column: 1;
-  -ms-grid-row-align: center;
   align-self: center;
   box-sizing: border-box;
-  grid-area: logo;
-  height: 100%;
-  width: 100%;
+  height: 20px;
```

Comment:
21px tall in zeplin.  Also this icon isn't as wide as it is tall so the `width` should probably be left unspecified

Reply:
> 21px tall in zeplin. Also this icon isn't as wide as it is tall so the `width` should probably be left unspecified

Moot bc we replaced the whole thing with an image as discussed in Slack.